cowamm/profitability/competitor_kpis/pancakeswap/pancakeswap_syncs_4232660.sql

```diff
@@ -0,0 +1,27 @@
+-- Finds the pancakeswap v2 pool address given tokens specified in query parameters (regardless of order)
```

Comment:
Can we please use consistent structure for all queries (ie the same way of specifying what parameters are needed and why as in the other queries in the repo)?